data/mods/Magiclysm/items/ethereal_items.json

```diff
@@ -1386,5 +1386,86 @@
     "color": "green",
     "flags": [ "TRADER_AVOID", "SINGLE_USE" ],
     "use_action": { "type": "heal", "bandages_power": 3, "bleed": 12, "move_cost": 300 }
+  },
+  {
+    "id": "technomancer_summoned_radio",
+    "type": "TOOL",
+    "name": { "str": "aetherspeaker (speaking)", "str_pl": "aetherspeakers (speaking)" },
+    "description": "A magically-summoned aetherspeaker, capable of communicating long-distance with other aetherspeakers or conventional radios.  With a command word, it may instead scan for radio signals.",
+    "weight": "295 g",
+    "volume": "125 ml",
+    "to_hit": -1,
+    "material": [ "concentrated_mana" ],
+    "symbol": ";",
+    "color": "cyan",
+    "flags": [ "TWO_WAY_RADIO", "ONLY_ONE", "TRADER_AVOID", "NO_DROP" ],
+    "use_action": [ { "type": "transform", "target": "technomancer_summoned_radio_on", "menu_text": "Turn to scanning mode" } ],
+    "melee_damage": { "bash": 4 }
+  },
+  {
+    "id": "technomancer_summoned_radio_on",
+    "copy-from": "technomancer_summoned_radio",
+    "type": "TOOL",
+    "name": { "str": "aetherspeaker (scanning)", "str_pl": "aetherspeakers (scanning)" },
+    "description": "An aetherspeaker tuned to listen for radio signals from the surrounding area.  With a command word, it may change frequency or switch to communicating with other aetherspeakers or radios.",
+    "use_action": [ "RADIO_ON", { "type": "transform", "target": "technomancer_summoned_radio", "menu_text": "Turn to speaking mode" } ],
+    "tick_action": [ "RADIO_TICK" ],
+    "flags": [ "TRADER_AVOID", "ONLY_ONE", "NO_DROP" ]
+  },
+  {
+    "id": "technomancer_summoned_ups",
+    "type": "MAGAZINE",
+    "category": "tool_magazine",
+    "name": { "str": "technomantic power supply" },
+    "description": "A smoothly-curved conglomeration of mana, humming softly as it provides power.",
+    "weight": "125 g",
+    "volume": "65 ml",
+    "material": [ "concentrated_mana" ],
+    "symbol": "=",
+    "color": "yellow",
+    "ammo_type": [ "battery" ],
+    "capacity": 1250,
+    "looks_like": "spiral_stone",
+    "flags": [ "TRADER_AVOID", "ONLY_ONE", "NO_DROP", "NO_SALVAGE", "NO_UNLOAD", "NO_RELOAD", "IS_UPS" ],
+    "pocket_data": [ { "pocket_type": "MAGAZINE", "rigid": true, "ammo_restriction": { "battery": 1 } } ],
+    "relic_data": { "charge_info": { "recharge_type": "periodic", "time": "2 s", "regenerate_ammo": true } },
+    "//": "Recharge time is 2 seconds due to bug #48019, making it actually 1 second. Reduce to 1 second if that ever gets fixed."
+  },
+  {
+    "id": "technomancer_welder",
+    "type": "TOOL",
+    "name": { "str": "voltaic touch" },
+    "description": "Your hands may currently spark with electricity powerful enough to heat metal to searing temperatures and weld it together.",
```

Comment:
No electric damage if you punch with it?

Reply:
...you know, I hadn't even thought of that, but you're right, _someone_ is going to try it.